src/main/java/duke/Event.java

```diff
@@ -0,0 +1,65 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Class to handle event task
+ */
+public class Event extends Task {
+    protected LocalDate start;

```

Comment:
Similar to `Deadline`, perhaps you can rename `start` as `startDate` and `end` as `endDate`